### PR TITLE
[TECH-549]Confirm page full changes

### PR DESCRIPTION
**Pull request by JeanMariaJose**

### CODE REVIEW

File: src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export interface Rate {
   currency: string
   ratePlan: string
   rateType: string
-  isCancellable: boolean
+  iscancellable: boolean
```
Comment (abhijithramankutty) [inline src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts:16]: better to keep it `isCancellable` it self.

File: src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx [hunk 2]
```diff
@@ -117,7 +119,7 @@ function RoomList({
 
                     <div className="flex ">
                       <IoIosCheckmark className="text-lg text-green-600 ml-2 mt-2"></IoIosCheckmark>
-                      {room.isCancellable ? (
+                      {room.iscancellable ? (
```
Comment (abhijithramankutty) [inline src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx:122]: revert to `isCancellable` it self w.r.t above comment

File: src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts (hunk 1 above)
Comment (JeanMariaJose) [reply src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts:16]: from backend its coming as iscancellable.  This was showing wrong in UI, after changing got corrected.. Or else we may have to change backend and proto

File: src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx (hunk 2 above)
Comment (JeanMariaJose) [reply src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx:122]: same as above

File: src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts (hunk 1 above)
Comment (abhijithramankutty) [reply src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts:16]: ok.fine

**Review by abhijithramankutty — APPROVED** [review]

**Pull request merged**